modules/nw-dual-stack-convert.adoc

```diff
@@ -33,36 +34,59 @@ After converting to dual-stack networking only newly created pods are assigned I
   path: /spec/serviceNetwork/-
   value: fd02::/112 <2>
 ----
-<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 CIDR prefix must be large enough to accommodate the specified host prefix.
-
+<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 Classless Inter-Domain Routing (CIDR) prefix must be large enough to accommodate the specified host prefix.
 <2> Specify an IPv6 CIDR with a prefix of `112`. Kubernetes uses only the lowest 16 bits. For a prefix of `112`, IP addresses are assigned from `112` to `128` bits.
---
 
-. To patch the cluster network configuration, enter the following command:
+. To patch the cluster network configuration, enter the following command in your CLI:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ oc patch network.config.openshift.io cluster \
+$ oc patch network.config.openshift.io cluster \ <1>
   --type='json' --patch-file <file>.yaml
 ----
+<1> Where `file` specifies the name of your created YAML file.
 +
---
-where:
+.Example output
+[source,text]
+----
+network.config.openshift.io/cluster patched
+----
 
-`file`:: Specifies the name of the file you created in the previous step.
---
+. To specify IPv6 Virtual IPs (VIPs) for API and Ingress services for your cluster, create a YAML configuration patch file that has a similar configuration to the following example file:
++
+[source,yaml]
+----
+- op: add
+  path: /spec/platformSpec/baremetal/machineNetworks/- <1>
+  value: fd2e:6f44:5dd8::/64
+- op: add
+  path: /spec/platformSpec/baremetal/apiServerInternalIPs/- <2>
+  value: fd2e:6f44:5dd8::4
+- op: add
+  path: /spec/platformSpec/baremetal/ingressIPs/-
+  value: fd2e:6f44:5dd8::5
+----
+<1> Ensure that you specify an address block for the `machineNetwork` network where your machines operate. You must select both API and Ingress IP addresses for the machine network.
+<2> Ensure that you specify each file path according to your platform. The example demonstrates a file path on a bare-metal platform.
+
+. To patch the cluster's infrastructure, enter the following command in your CLI:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc patch infrastructure cluster \ <1>
+  --type='json' --patch-file <file>.yaml
+----
+<1> Where `file` specifies the name of your created YAML file.
 +
 .Example output
 [source,text]
 ----
-network.config.openshift.io/cluster patched
+infrastructure/cluster patched
 ----
 
 .Verification
```

Comment:
Do these two verification steps have to be done in order? If the order doesn't matter, switch the ordered list to a bulleted list :)

Reply:
Yes, they are separate because the patch commands are different.